Readme.md

```diff
@@ -1,29 +1,46 @@
 
 # Customize WebVI with CSS
+This example demonstrates how to customize the HTML output of a WebVI using CSS. 
+
+One of the main parts of a WebVI is an HTML output that displays HTML5 Custom Elements. One of the advantages of using CSS in a web application is to the separate content from style. For example, you could create a custom CSS file that is shared across the organization or even among other users to build more and more styles and layouts.
 
 # Embed Content into a WebVI
+This example demonstrates how to embed custom web content into the WebVI panel using LabVIEW NXG 2.0 Beta. WebVIs use HTML to define and describe the content of the panel that is loaded in a web page. This means that you can add custom HTML content to appear alongside the LabVIEW-generated HTML.
 
 # Embed a WebVI into Web Content
-This example demonstrates embedding a Web VI built using LabVIEW NXG 2.0 Beta into an entirely different static Web page. WebVIs are composed of three basic parts: HTML5 Custom Elements which compose the view of the page, the WebVI block diagram (.via.txt) which provides the client side logic, and the JavaScript and CSS files that make the Web application run. Because these are the same basic building blocks (san via.txt) of all other Web pages you can pull the Web VI apart and embed it with any Web content.
+This example demonstrates how to embed the output of a WebVI built using LabVIEW NXG 2.0 Beta into an static web page. 
+
+The build output of a WebVI includes three basic parts: 
+- HTML5 custom elements
+- Compiled WebVI diagram (`.via.txt`)
+- JavaScript and CSS files used in the web application 
+
+Because WebVIs share the same basic building blocks as other web pages, you can embed WebVI output into any web content.
 
 # Incorporate User Resource into WebVI
+This example demonstrates how to add resource files such as images, CSS files, JavaScript files, and HTML files to your web application component and reference them in a WebVI without needing to upload these files to a web server.
+
+You can also use resource files to augment the WebVI's capabilities. For example, you can add other HTML widgets or scripts and reference them in the HTML view of the WebVI.
 
-<!-- The following should be equivalent to the section in webvi-examples/MultipleTopLevelWebVIs/Readme.md -->
-# Multiple Top-Level WebVIs
-[![Multiple Top-Level WebVIs Demo Link](https://img.shields.io/badge/Details-Demo_Link-green.svg)](https://ni.github.io/webvi-examples/MultipleTopLevelWebVIs/Builds/Web%20Server/Configuration1/MultipleTopLevelWebVIs/)
-[![Multiple Top-Level WebVIs README Link](https://img.shields.io/badge/Details-README_Link-orange.svg)](https://github.com/ni/webvi-examples/tree/master/MultipleTopLevelWebVIs)
+<!-- The following should be equivalent to the section in webvi-examples/MultipleTopLevelVIs/Readme.md -->
+# Multiple Top-Level VIs
```

Comment:
You may need to re-copy and paste the header and summary sections from the full README here to get the VI -> WebVI changes